### PR TITLE
Bug 1564122 - update to newest md-directory (including non-vulnerable…

### PR DESCRIPTION
**Pull request by djmitche**
… dependencies)

Bugzilla Bug: [1564122](https://bugzilla.mozilla.org/show_bug.cgi?id=1564122)

### CODE REVIEW

File: infrastructure/builder/src/generate/generators/docs-tocs.js [hunk 1]
```diff
@@ -152,14 +152,19 @@ exports.tasks = [{
   requires: [],
   provides: ['docs-toc'],
   run: async (requirements, utils) => {
-    const files = await mdParseDir(DOCS_DIR, { dirnames: true });
+    const filesWithExtensions = await mdParseDir(DOCS_DIR, { dirnames: true });
+    // strip .md and .mdx extensions frmo those filenames..
+    const files = Object.assign(
+      ...Object.entries(filesWithExtensions)
+        .map(([filename, value]) => ({[filename.replace(/\.mdx?/, '')]: value})));
+    console.log(Object.keys(files));
```
Comment (owlishDeveloper) [inline infrastructure/builder/src/generate/generators/docs-tocs.js]: `console.log` ?

Comment (djmitche) [reply infrastructure/builder/src/generate/generators/docs-tocs.js]: Oh, thanks, I had deleted but not committed the change.

@helfi92 can you double-check me in thinking that this is unused in the UI?

**Comment by djmitche** [comment]
Oh, dear, this still isn't right - the docs TOC is all manner of messed up.

**Comment by djmitche** [comment]
Hm, it's outputting bogus filenames:

```
--- before.json 2019-07-10 16:42:47.144505693 +0000
+++ after.json  2019-07-10 16:41:49.989288811 +0000
@@ -1,677 +1,862 @@
 {
-  "manual/design/apis/errors": {
+  "manual/design/apis/manual/design/apis/errors": {
```

**Comment by djmitche** [comment]
https://github.com/s3ththompson/md-directory/issues/2

**Pull request closed without merging**